website/src/layouts/Layout.astro

```diff
@@ -1,56 +0,0 @@
----
-import { Styles } from "@gemini-ui-astro/styles";
-import { DynamicGlow } from "@gemini-ui-astro/components";
-
-interface Props {
-  title: string;
-}
-
-const { title } = Astro.props;
----
-
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="description" content="Astro description" />
-    <meta name="viewport" content="width=device-width" />
-    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
-    <meta name="generator" content={Astro.generator} />
-    <title>{title}</title>
-    <DynamicGlow />
```

Comment:
I don't see where this is being imported in the docs website but it seems to be working anyway somehow 🤔 any idea how this is working?

Reply:
I have no idea actually